worlds/ladx/Options.py

```diff
@@ -40,49 +53,41 @@ class Logic(Choice, LADXROption):
 
 class TradeQuest(DefaultOffToggle, LADXROption):
     """
-    [On] adds the trade items to the pool (the trade locations will always be local items)
-    [Off] (default) doesn't add them
+    Trade quest items are randomized. Each NPC takes its normal trade quest
+    item, but gives a random item.

```

Comment:
```suggestion
    item and gives a randomized item in return.
```
I think it's a little clearer this way.